index.html

```diff
@@ -101,7 +101,8 @@ <h3>
     <p>
       The {{RTCPeerConnection}}
       interface is defined in [[WEBRTC]]. This document extends that interface
-      by adding an additional static method.
+      by adding an additional static method, and adding a Content-Security-Policy
```

Comment:
I don't see the additional static method in this PR. Did it get deleted again?

Reply:
This is referring to getDefaultIceServers(), which is added by -extensions, but not by this pr -- only the  ", and adding a Content-Security-Policy" part of this line is new.